_guidelines-v4/10-analysisharm/01-analysis.md

```diff
@@ -6,4 +6,6 @@ title: "Analytical Information"
 sectionid: "analysis"
 ---
 
-This chapter describes the use of attributes that capture data which may be useful for analytical purposes. The analysis module provides attributes that record relationships between entities found in the encoding. These attributes may be used differently by different users, depending on the purpose of the analysis. These Guidelines recommend that encoders employ commonly accepted analytical practices, such as "functional analysis" or "Schenkerian analysis", and document their use in the {% include link elem="encodingDesc" %} described in section {% include link id="headerEncodingDescription" %}. For general information on musical analysis, please consult Grove Music Online, "[Analysis](https://doi.org/10.1093/gmo/9781561592630.article.41862){:.link_ref}".
+This chapter describes the use of attributes that capture data which may be useful for analytical purposes. The analysis module provides attributes that record relationships between entities found in the encoding. These attributes may be used differently by different users, depending on the purpose of the analysis.
+
+These Guidelines recommend that encoders employ commonly accepted analytical practices, such as ‘functional analysis’ or ‘Schenkerian analysis’, and document their use in the {% include link elem="encodingDesc" %} described in section {% include link id="headerEncodingDescription" %}. For general information on musical analysis, please consult Grove Music Online, ‘[Analysis](https://doi.org/10.1093/gmo/9781561592630.article.41862){:.link_ref}’.
```

Comment:
Memo for later: We should find an external link to musical analysis that is not behind a subscription wall.

Reply:
may we turn that into a separate issue and assign it to you? ;-)

Reply:
Yes, please, but I do not insist on the assignment.